src/log.py

```diff
@@ -0,0 +1,44 @@
+import bs4
+import requests
+
+#create a log class, that will parse the log file and return the info
```

Comment:
This isn't a log class

Reply:
yep, that was a github copilot comment oops